packages/react-components/src/organisms/TeamsTabbedCard.tsx

```diff
@@ -29,32 +30,31 @@ type TeamsTabbedCardProps = Pick<
   ComponentProps<typeof TabbedCard>,
   'title' | 'description'
 > & {
-  teams: ReadonlyArray<Pick<TeamResponse, 'id' | 'displayName'>>;
+  teams: ReadonlyArray<
+    Pick<TeamResponse, 'id' | 'displayName' | 'inactiveSince'>
+  >;
+  disableActiveTab?: boolean;
```

Comment:
I think this raises issues about the disabled requirement. What happens if we have an active group/team/whatever with only alumni members? You would default to a disabled tab with no content which is a very odd behaviour

Reply:
I'll add a 'There are no past leaders.' paragraph when the tab is empty